src/components/POLine/Item/ContributorForm.js

```diff
@@ -48,13 +49,54 @@ class ContributorForm extends Component {
               </div>
             </Col>
           )}
-          {fields.map(this.renderSubForm)}
```

Comment:
with react-final-form such construction fails, since no `fields` is passed to render function. So I've moved code of render function inline